kolibri/plugins/coach/assets/src/views/class-list-page/index.vue

```diff
@@ -37,7 +37,7 @@
 <script>
 
   import * as constants from '../../constants';
-  import orderBy from 'lodash/orderBy';
+  import { orderBy } from 'lodash/orderBy';
```

Comment:
Does this make a difference to the import?

Reply:
Not entirely sure, but it optimized the setup wizard module by a lot, so I stuck to this convention.